src/core_plugins/tests_bundle/tests_entry_template.js

```diff
@@ -26,7 +26,7 @@ window.__KBN__ = {
   vars: {
     kbnIndex: '.kibana',
     esShardTimeout: 1500,
-    esApiVersion: '2.0',
+    esApiVersion: 'master',
```

Comment:
I'm pretty sure it will still complain, but I'm down. Then we can have a versions file separate from package.json